python/paddle/fluid/contrib/slim/tests/test_imperative_qat.py

```diff
@@ -275,7 +275,8 @@ def _build_static_lenet(main, startup, is_test=False, seed=1000):
 
         reader = paddle.batch(
             paddle.dataset.mnist.test(), batch_size=32, drop_last=True)
-        weight_quantize_type = 'abs_max'
+        #weight_quantize_type = 'abs_max'
+        weight_quantize_type = 'channel_wise_abs_max'
```

Comment:
Don't modify this file, please add your own unit test.

Reply:
Fixed